webapp/src/app.tsx

```diff
@@ -2853,7 +2853,11 @@ export class ProjectView
     }
 
     private editorLoaded() {
-        pxt.tickEvent('app.editor', { projectHeaderId: this.state.header?.id });
+        pxt.tickEvent('app.editor', {
+            projectHeaderId: this.state.header?.id,
+            fileType: this.editorFile?.getExtension(),
+            accessibleBlocks: this.getData<boolean>(auth.ACCESSIBLE_BLOCKS) ? "true" : "false"
```

Comment:
Do we need to send accessible Blocks or not on every new project creation?

Reply:
Rest looks good.

Reply:
I think new tick is preferable, it seems weird to add custom dimensions to other tick.

Reply:
@abchatra gotcha, i'm going to leave the file extension dimension here though since this seems like the best place for it